dev-proxy/index.ts

```diff
@@ -0,0 +1,54 @@
+import fetch from "node-fetch"
+import {launchChrome} from "./launch-chrome.js";
+import * as mockttp from "mockttp"
+// https://httptoolkit.github.io/mockttp/interfaces/Mockttp.html#forGet
+// https://github.com/httptoolkit/mockttp/tree/main
+
+(async () => {
+    if (process.argv.length < 4) {
+        console.error("You need to provide the name of the instance as the second parameter");
+        process.exit(-1);
+    }
+    const instance = process.argv[2]
+    const scriptFile = process.argv[3]
+    
+    // Create a proxy server with a self-signed HTTPS CA certificate:
+    const https = await mockttp.generateCACertificate();
+    const server = mockttp.getLocal({ https, debug: false });
+
+    const target = `${instance}.matrixreq.com`
+    const url = `https://${target}`
+
+    const addedUrl = `<script src="/${scriptFile}"></script>`
+    const replaced = `${addedUrl}\n</body>\n</html>`
+    const page = await fetch(url).then(data => data.text())
+    const patchedPage = page.replace(/<\/body>\s*<\/html>/, replaced)
+
+    await server.forAnyWebSocket().always()
+        .thenPassThrough();
+    await server.forGet(url).always()
```

Comment:
if you refresh a URL with http://myInstance.matrixreq.com/PROJECT/ITEM
it doens't get included... Would it work with regex ? (Also for adminConfig)